aepsych/models/variational_gp.py

```diff
@@ -115,3 +118,70 @@ def get_config_options(cls, config: Config, name: Optional[str] = None):
         if options["likelihood"] is None:
             options["likelihood"] = BernoulliLikelihood()
         return options
+
+
+class OrdinalGP(VariationalGP):
+    """
+    Convenience class for using a VariationalGP with an OrdinalLikelihood.
+    """
+
+    outcome_type = "ordinal"
+    classname = "OrdinalGP"
+
+    def predict(self, xgrid, probability_space=False):
+        with torch.no_grad():
+            post = self.posterior(xgrid)
+        fmean = post.mean.squeeze()
+        fvar = post.variance.squeeze()
+
+        if probability_space:
```

Comment:
Is everything below here just copying `predict_probs` from `OrdinalGPModel`? Any way to make this less copy-pastey and ugly?

Reply:
what are your thoughts on the newer approach